src/components/PageElements/Elements/ReviewResponseElement.tsx

```diff
@@ -1,39 +1,41 @@
 import React from 'react'
-import { Icon, Grid } from 'semantic-ui-react'
+import { Icon } from 'semantic-ui-react'
 import { ReviewResponse } from '../../../utils/generated/graphql'
 import { ElementDecisionLabel } from '../../Review'
 
 interface ReviewResponseElementProps {
   isCurrentReview: boolean
   isConsolidation: boolean
+  isDecisionVisible?: boolean
   shouldDim?: boolean
-  shouldHideDecision?: boolean
   reviewResponse: ReviewResponse
 }
 
 const ReviewResponseElement: React.FC<ReviewResponseElementProps> = ({
   isCurrentReview,
   isConsolidation,
-  reviewResponse,
+  isDecisionVisible = true,
   shouldDim = false,
+  reviewResponse,
   children,
-  shouldHideDecision = false,
 }) => {
   const backgroundClass = isCurrentReview ? 'changeable-background' : ''
   const dimClass = shouldDim ? 'dim' : ''
 
+  const canReviewerEdit = (isCurrentReview && !!reviewResponse?.comment) as boolean
+
   return (
     <div className={`response-container ${backgroundClass} ${dimClass}`}>
       <div className="review-response-content ">
-        {!shouldHideDecision && (
-          <ElementDecisionLabel isConsolidation={isConsolidation} reviewResponse={reviewResponse} />
-        )}
-        {!reviewResponse.comment ? null : (
-          <div className="comment-container">
-            <Icon name="comment alternate outline" color="grey" />
-            <p className="secondary">{reviewResponse.comment}</p>
-          </div>
-        )}
+        <ElementDecisionLabel
```

Comment:
hmm was wondering why we would add an extra prop and extra changes to change from shouldHideDecision to isDecisionVisible ?

Reply:
The only difference I can see is showing date of review. But it's not actually shown in the summary view of change requests for applicant (based on Miro)

Reply:
I think the date is a nice one to have - like is used in some other places that aren't part of designs either.
Also changed because thought the code looks simples to read, by moving this logic to be inside.